### PR TITLE
Fix broken views functionality

### LINKED ISSUE #3: Views integration not working
> The one main thing that wasn't working for me was when creating a View. I attempted some debugging but stopped because of other priorities.

(Pasted from [a comment in another thread](https://github.com/backdrop-contrib/search_api/issues/1#issuecomment-622431363))

### PR DESCRIPTION
**Pull request by earlyburg**
Fixes https://github.com/backdrop-contrib/search_api/issues/3

### CODE REVIEW

File: contrib/search_api_views/search_api_views.info [hunk 1]
```diff
@@ -1,9 +1,7 @@
 name = Search Views
 description = Integrates the Search API with Views, enabling users to create views with searches as filters or arguments.
 dependencies[] = search_api:search_api
-dependencies[] = entity_plus:entity_plus
+dependencies[] = entity_plus (>1.x-1.0.10)
```
Comment (laryn) [inline contrib/search_api_views/search_api_views.info]: Haven't tested but should this be `>=`?

Comment (earlyburg) [reply contrib/search_api_views/search_api_views.info:4]: I used this page as a source:
https://api.backdropcms.org/modules

The dependencies[] property in the .info file can also optionally specify versions:

dependencies[] = exampleapi (>1.0)
The above module requires any version greater than version 1.0.

You can optionally specify the core version number as well:
dependencies[] = exampleapi (>1.x-1.5.0)

The above module requires a 1.x version compatible version of the module and a version greater than 1.5.

Additionally, multiple version dependencies can be specified as comma-separated values within the parentheses:

dependencies[] = exampleapi (>1.0, <=3.2, !=3.0)

Comment (earlyburg) [reply contrib/search_api_views/search_api_views.info:4]: Oh duh I see

Comment (earlyburg) [reply contrib/search_api_views/search_api_views.info]: This has been done

**Comment by earlyburg** [comment]
@laryn Hello do you see any reason why we should not merge this? @docwilmot does not care if I take over this project.
https://github.com/backdrop-contrib/search_api/issues/10
 Thank you.

**Comment by laryn** [comment]
@earlyburg It looks like you are a maintainer on this one at this point! Go for it.

**Comment by earlyburg** [comment]
@laryn Github tells me that only those with write access to this repository can merge pull requests.
Additionaly, an attempt to push code from my remote repo to this one returned a "permission denied/ 403" error.
Looks like although I have been listed as a maintainer on the info file, I'm more locked out of this github repo than I am for the entity_plus module. When I submit a pull request for entity_plus module I have access to the "merge this pull request" button, but for this repo I seem to lack appropriate permissions. 
Sorry for the trouble but this appears to be something that requires further assistance. Thank you.

**Comment by laryn** [comment]
@earlyburg Sometimes the Github workflow needs a little manual grease... I added this to the `-ops` queue and I think @klonos is the one who will adjust permissions accordingly:

 - https://github.com/backdrop-ops/contrib/issues/464

**Comment by earlyburg** [comment]
@laryn No worries, I really appreciate it.

**Comment by klonos** [comment]
@earlyburg I've added you to the repo with "maintain" access. Can you please check if you have enough permissions to manage the project?

**Comment by earlyburg** [comment]
@klonos Hello, I have been able to merge my pull requests for this project, and in addition, I was able to push and pull from my remote repo to this one. Thank you very much for your assistance, I appreciate it.

**Pull request merged**